code-challenges.test.js

```diff
@@ -49,8 +71,27 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+//  describe("firstLetter", () => {
+//       it("takes in an array of words and returns an array with all the words capitalized", () => {
+//           expect(firstLetter(["streetlamp", "potato", "teeth", "conclusion", "nephew"]).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]))
+//           (firstLetter(["temperature", "database", "chopsticks", "mango", "deduction"]).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]))
+//       })
+//   })
+
+//   error code return
+//   b) Create the function that makes the test pass.
+
+// Pseudocode
+// Create a function called firstLetter that takes the two given arrays and changes the first letter of each string in the array and returns an Uppercase letter by creating a variable within that takes the index of each word in the array and changes the lower case letter toUppercase.
```

Comment:
Your intent in the pseudocode is heading in the right direction, there may be some syntax that needs adjusting to achieve the desired results.